chainer/training/triggers/interval.py

```diff
@@ -40,9 +42,16 @@ def __call__(self, trainer):
         """
         updater = trainer.updater
         if self.unit == 'epoch':
-            prev = self.count
-            self.count = updater.epoch_detail // self.period
-            return prev != self.count
+            epoch_detail = updater.epoch_detail
+            previous_epoch_detail = updater.previous_epoch_detail
```

Comment:
> In an extension I'm developing, it holds "logical" variable and function nodes(*) and triggers associated with them. Not necessarily all logical nodes are instantiated in each iteration, because of Define-by-Run nature. Only when the associated logical nodes are instantiated, the extension calls these triggers with the trainer instance (which was given to the extension).

Thank you. This is exactly what I want to know.